.cirrus.yml

```diff
@@ -87,6 +87,34 @@ qa_task:
     - jfrog rt npm-config --repo-resolve npm --repo-deploy sonarsource-npm-public-qa
     - git submodule update --init --depth 1 -r
     - jfrog rt npm-ci
+    - npm install eslint-ruling@npm:eslint@8.57.0
```

Comment:
this should be a no-op, but just to be explicit, let us make sure it is running eslint8